README.md

```diff
@@ -44,11 +39,6 @@ Feel free to check the [issues page](../../issues/).
 Give a ⭐️ if you like this project!
 
 
-## Acknowledgments
```

Comment:
- [ ] As it is a requirement for this project and best practice, please kindly acknowledge the author of your design.